contracts/remappings.txt

```diff
@@ -5,3 +5,5 @@ forge-std/=foundry-lib/forge-std/src/
 hardhat/=node_modules/hardhat/
 @eth-optimism/=node_modules/@eth-optimism/
 @scroll-tech/=node_modules/@scroll-tech/
+solady/=node_modules/solady
+@rari-capital/=node_modules/@rari-capital/
```

Comment:
I do see a @rari-capital utility file being vendored below. Is this remapping correct?

Reply:
TBH I'm not sure if we need this or not and where are these remappings applied. I can try to remove it and see what happens.

Reply:
Actually, the easiest way to not deal with all these dependencies is to just create an interface out of the `GasPriceOracle` contract since we are just calling the predeploy contract. :joy:

The implementation is just a good to have to understand what's going on under the hood, but might not even be the actual contract deployed.

Reply:
What @leeyikjiun is saying is correct - our contract code doesn't actually need any of these extra dependencies, it's just a convenient way to have access to all the code for the `GasPriceOracle` contract.

So if I vendor them and remove changes from `package.json` and `native_solc_compile, would that be sufficient to help us prevent any accidental bytecode changes in the future?

Reply:
rari-capital has been removed from package.json and remappings.txt.